pkg/ksync/export_test.go

```diff
@@ -0,0 +1,3 @@
+package ksync
```

Comment:
What is this file about? It is marked as a test, but no tests are defined in here. Also if it were a test file it needs to be classified as either _unit_ or _integration_ test. This causes the linting task to fail atm.

Reply:
The idea was to test functions which are unexported, and this is the standard technique to test it. It is also common in the go standard library. Source: https://medium.com/@robiplus/golang-trick-export-for-test-aa16cbd7b8cd